### ISSUE TITLE
problem 2 hyperparameter

### ISSUE THREAD

**Issue by cnimativ** [jermwatt/RNN_demo#22]
the network trains, but the quality of the hyperparameters setup is really bad. boosted my batch size and increase epoch to make testing go faster. the training loss is really bad at 2.3. given that this is just a beta test, not going to put in work to optimize the hyperparams. ill send my notebook to mike.
